### ISSUE TITLE
Google Translate is automatically translating text that should not be translated

### ISSUE THREAD

**Issue by KlausGPaul** [open-apparel-registry/open-apparel-registry#2370]
## Overview

When selecting a non-english language option from the google translate bar, the content of the page gets translated automatically. For the landing page, the term "OS" is considered to be the abbreviation for "Operating System", e.g.
* Nome struttura o ID sistema operativo
* Einrichtungsname oder Betriebssystem-ID
* Nombre de la instalación o ID del sistema operativo
* Tesis Adı veya İşletim Sistemi Kimliği

![image](https://user-images.githubusercontent.com/20340525/220315782-2d23ffdc-c274-4175-9bce-5d1effb8be9c.png)

Note this also affect the display of the search results, both company names and OS-IDs

![image](https://user-images.githubusercontent.com/20340525/220315288-5ab25c3e-03a2-4ef6-a382-5ae50169fb60.png)

and portions of addresses (lot -> Grundstück)

![image](https://user-images.githubusercontent.com/20340525/220315635-1e57c022-3f8c-49d5-8c05-519492aefce0.png)

There is another unwanted behaviour when e.g. being on a translated page such as https://opensupplyhub.org/facilities . For the field "Product Type", the blind text is translated (when using German it would read "z.B. Jacken"), while the search will positively not find Jacken, only jackets 

![image](https://user-images.githubusercontent.com/20340525/220315983-d1f4b30e-7777-4bc5-9ea5-fba849905241.png)

### Expected Behavior

Certain terms should not be translated

### Actual Behavior

OS ID gets translated such that the resulting text makes absolutely no more sense, whilst the OS ID is the key feature of the offering.

### Steps to Reproduce

* Open https://opensupplyhub.org
* Select, e.g., German

### Demo

N/A 

### Additional context

This error did not occur on openapparel as OS ID used to be called OAR ID.

**Comment by jwalgran** [comment]
From the Google Cloud Translation documentation https://cloud.google.com/translate/troubleshooting

>How do I tell Cloud Translation - Advanced to not translate something?
>You can use the following HTML tags:
>
```
<span translate="no"> </span>
<span class="notranslate"> </span>
```
>
>This functionality requires the source text to be submitted in HTML.

**Comment by jwalgran** [comment]
We already use `notranslate` in a few places.  Proposed solution is to wrap all usages of `OS ID` with a `notranslate` style.

**Comment by jwalgran** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Civic-Apps-5c8fabc1177a1c1f6672f901/issues/open-apparel-registry/open-apparel-registry/2370?planning-poker) with Zenhub @mstone121 @TaiWilkin

**Comment by mariel-oar** [comment]
@jwalgran , does this estimate include wrapping the example search texts (e.g. "select" and "jackets") in a no translate style?

**Comment by jwalgran** [comment]
@mariel-oar Yes, the estimate does cover all of the undesired translations in the issue description. I updated the issue title to cover all cases rather than specifically calling out OS ID.
